pytest_pyodide/hook.py

```diff
@@ -172,19 +200,55 @@ def _get_item_position(item):
 
 
 @pytest.hookimpl(tryfirst=True)
-def pytest_runtest_setup(item):
-    if not hasattr(item, "fixturenames"):
-        # Some items like DoctestItem has no fixture
-        return
+def pytest_collection(session):
+    return None
+
 
-    if not pytest.pyodide_runtimes and "runtime" in item.fixturenames:  # type: ignore[truthy-bool]
-        pytest.skip(reason="Non-host test")
-    elif not pytest.pyodide_run_host_test and "runtime" not in item.fixturenames:  # type: ignore[truthy-bool]
-        pytest.skip("Host test")
+@pytest.hookimpl(tryfirst=True)
+def pytest_runtest_setup(item):
+    if item.config.option.run_in_pyodide:
+        if not hasattr(item, "fixturenames"):
+            return
+        if pytest.pyodide_runtimes and "runtime" in item.fixturenames:  # type: ignore[truthy-bool]
+            pytest.skip(reason="pyodide specific test, can't run in pyodide")
+        else:
+            # pass this test to pyodide runner
+            # first: make sure that pyodide has the current folder copied over
+            itemPath = Path(item.path)
+            copy_files = list(itemPath.parent.glob("**/*"))
```

Comment:
What if there are subdirectories? This should probably recursively list the files no?

Reply:
It does, it's a pathlib.path, glob is recursive by default